setup.py

```diff
@@ -12,76 +22,104 @@
     subprocess.check_call([sys.executable, '-m', 'pip', 'install', 'dunamai'])
     from dunamai import Version
 
+try:
+    from packaging.version import parse as package_parse
+except ImportError:
+    from pip._vendor.packaging.version import parse as package_parse
+
+
+def latest_pypi_version(package: str, prerelease: bool = False) -> str:
+    """ https://stackoverflow.com/a/34366589 """
+    response = requests.get(f'https://pypi.python.org/pypi/{package}/json')
+
+    if response.status_code == requests.codes.ok:
+        releases = json.loads(response.text).get('releases', [])
+        versions = list(sorted(package_parse(release) for release in releases))
+        if not prerelease:
+            versions = [version for version in versions
+                        if not version.is_prerelease]
+        version = str(max(versions))
+    else:
+        version = None
+    return version
```

Comment:
new function to retrieve the latest version from the list of releases given by the PyPI API